vars/capsuleSystemTests.groovy

```diff
@@ -0,0 +1,196 @@
+/* groovylint-disable 
+  BuilderMethodWithSideEffects, CompileStatic, DuplicateStringLiteral, 
+  FactoryMethodName, VariableTypeRequired */
+
+
+void gitClone(String repository, String branch, String directory, String credentialsId) {
+  dir(directory) {
+    checkout([
+      $class: 'GitSCM',
+      branches: [[name: branch]],
+      userRemoteConfigs: [[
+          url: 'git@github.com:vegaprotocol/' + repository + '.git',
+          credentialsId: credentialsId
+    ]]])
+  }
+}
+
+void buildGoBinary(String directory, String outputBinary, String packages) {
+  dir(directory) {
+    sh 'go mod vendor'
+    sh 'go build -o ' + outputBinary + ' ' + packages
+  }
+}
+
+void trysh(String shellScript) {
+  try {
+    sh shellScript
+  } catch(err) {
+  }
+}
+
+void cleanup(String testDirectoryPath) {
+  trysh 'rm ' + testDirectoryPath + '/vegacapsule'
+  trysh 'rm ' + testDirectoryPath + '/vega'
+  trysh 'rm ' + testDirectoryPath + '/data-node'
+  trysh 'rm -r ' + testDirectoryPath + '/testnet'
+  trysh 'rm ' + testDirectoryPath + '/vegawallet'
+  trysh 'rm ' + testDirectoryPath + '/config_system_tests.hcl'
+}
```

Comment:
Do we need this function? What is a difference compared with `cleanWs()`? 
https://www.jenkins.io/doc/pipeline/steps/ws-cleanup/

Reply:
No, you are correct. I did not know this function but it's obviously better :)